plugin/pymode.vim

```diff
@@ -219,56 +221,57 @@ if g:pymode_rope
     call pymode#default('g:pymode_rope_completion_bind', '<C-Space>')
```

Comment:
Should we use the prefix here somehow?
What if we have `rope_completion_bind` as a variable too, with default to `<C-space>`?

Because that's the only command without the prefix, so it may be nice to have it explicitly defined.

Reply:
IMHO I'd leave it as-is. It's the only command to use C-Space and the help docs describe how to remap it.